Containerfile

```diff
@@ -1,35 +1,38 @@
-ARG IMAGE_NAME="${IMAGE_NAME:-silverblue}"
-ARG SOURCE_IMAGE="${SOURCE_IMAGE:-silverblue}"
-ARG SOURCE_ORG="${SOURCE_ORG:-fedora-ostree-desktops}"
+ARG IMAGE_NAME=silverblue
+ARG SOURCE_IMAGE=silverblue
+ARG SOURCE_ORG=fedora-ostree-desktops
 ARG BASE_IMAGE="quay.io/${SOURCE_ORG}/${SOURCE_IMAGE}"
-ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION:-40}"
+ARG FEDORA_MAJOR_VERSION=40
+
+# workaround for selinux denying direct access to mounted buildcontext
+FROM scratch as context
+COPY / /
```

Comment:
Does this workaround address the problem with `tar` failing?

Reply:
No, do you have a reference or example for that issue?  
Initially i tried bind mounting the buildcontext directly (by omitting `from=`), but access from within the container was denied on selinux enabled podman machine executors.

Reply:
This is only for SELinux handling for copying the build context into the container.

When mounting the actual build context instead doing this scratch workaround.... selinux yells at us and doesn't let us have access to the fiel.

Reply:
Oops, yes, @m2Giles answered for me... I missed the question. 

This part looks good to me.